assets/src/blocks/GuestBook/GuestBook.js

```diff
@@ -0,0 +1,15 @@
+export const GuestBook = () => {
+  const buildURL = () => {
+    const hostname = window.location.hostname;
+    const pathname = window.location.pathname.slice(1);
```

Comment:
One thing we need to think here, is that this is generated on the editor, so the `pathname` is containing `wp-admin`.

For instance, if you check a story on [this page|https://www-dev.greenpeace.org/test-pandora/story-map/], pick one of the stories and click on the copy icon just to get the share url. It's `https://www.greenpeace.org/test-pandora/wp-admin/post-new.php?id=1`

Reply:
I think this issue is solved now! 🙂